src/KafkaFlow/Consumers/IOffsetCommitter.cs

```diff
@@ -5,6 +5,6 @@ namespace KafkaFlow.Consumers
 
     internal interface IOffsetCommitter : IDisposable
     {
-        void StoreOffset(TopicPartitionOffset tpo);
+        void Commit(TopicPartitionOffset tpo);
```

Comment:
I would change this name because we are not committing here. Maybe EnqueueToCommit()